crates/sui-e2e-tests/tests/shared_objects_tests.rs

```diff
@@ -41,6 +41,39 @@ async fn shared_object_transaction() {
         .await;
 }
 
+/// Delete a shared object as the object owner
+#[sim_test]
+async fn shared_object_deletion() {
```

Comment:
We should also add a stress test in the e2e tests, where we generate a few hundreds of certs that try to delete the same shared object, and send them at once to the network, and observe how they behave. After they all finish, we then start a new fullnode to catch up from genesis to see if there is any issue with replay.

Reply:
Just added a test below this `shared_object_deletion_multiple_times` which creates 300 deletion transactions and then sends them all in at once, and then starts up a fullnode and waits for the synch from genesis.